cvat/apps/documentation/user_guide.md

```diff
@@ -24,97 +24,157 @@ There you can:
+
+__Select files__. Push this button to select files you want to annotate.
+
+Push ``Submit`` button and it will be added into the list of annotation tasks. Then, the created task will be displayed on dashboard:
+
+![](static/documentation/images/image006.jpg)
+
+3. The Dashboard contains elements and each of them relates to a separate task. They are sorted in creation order. Each element contains: task name, preview, execution status, buttons, and one or more links. Each button is responsible for a specific function:
+- ``Dump Annotation`` — download an annotation file from the task (xml format) 
+- ``Upload Annotation`` — uploading an annotation file to the task (xml format)
+- ``Update Task`` — bring up "Update task" panel. It is used to edit or add labels line
+- ``Delete Task`` — delete the task
+- ``Git Repository Sync`` — sync annotation with the repository. Presence depends on task configuration
+- ``Run TF Annotation`` — automatic annotation with Tensorflow Object Detection API. Presence depends on task configuration
+- ``Run Auto Annotation`` — automatic annotation with  OpenVINO toolkit. Presence depends on task configuration
+
+  Item color depends on status of synchronization with the repository: ``red`` means a task is not synchronized with the repository, 
+``yellow`` means a task is in a temporary branch of the repository, ``green`` means a task is merged into the repository.
+
+4. Follow a link inside ``Jobs`` section to start annotation process. In some cases, you can have several links. It depends on size of your task and ``Overlap Size`` and ``Segment Size`` parameters. To improve UX, only the first several frames will be loaded and you will be able to annotate first images. Other frames will be loaded in background.
 
     ![](static/documentation/images/image007.jpg)
 
+
+### Creating a task using model manager
+
+The application will be enabled automatically if [OpenVINO™ component](https://github.com/opencv/cvat/blob/develop/components/openvino) is installed. It allows to use custom models for auto annotation. Only models in OpenVINO™ toolkit format are supported. If you would like to annotate a task with a custom model, please convert it to the intermediate representation (IR) format via the model optimizer tool. See [OpenVINO documentation](https://software.intel.com/en-us/articles/OpenVINO-InferEngine) for details. To create a task using the model manager, click the model manager button on the main page
+
+![](static/documentation/images/image099.jpg)
+
+Model manager allows you to upload a model to form a pre annotation, for example, if there's no annotation. In the model manager window you can download or update models. [Read more](https://github.com/opencv/cvat/tree/develop/cvat/apps/auto_annotation)
```

Comment:
I don't understand the sentence. Should be something like the text below:

The model manager allows you to manage your deep learning (DL) models uploaded for auto annotation. Using the functionality you can upload, update or delete a specific DL model. Use "Auto annotation" button to pre annotate a task using one of your DL models.